src/components/SideNavigationBar/SideNavigationBar.tsx

```diff
@@ -1,34 +1,39 @@
 import { FC } from 'react'
 import { BiFile } from 'react-icons/bi'
 import { v4 as uuidv4 } from 'uuid'
+import { Link } from 'react-router-dom'
 
 const ListItems = [
   {
     id: uuidv4(),
     item: 'My Reports',
+    url: '/my-reports',
   },
   {
     id: uuidv4(),
     item: 'Analytics',
+    url: '',
```

Comment:
url is empty here?

Reply:
the analytic page is not ready yet, so i put a empty string here